src/Middleware/JsonSchema.php

```diff
@@ -74,4 +81,13 @@ private function normalizeFilePath($path)
 
         return 'file://'.$path;
     }
+
+    /**
+     * @param callable $errorHandler
```

Comment:
Recommend including a description and what signature the errorHandler needs to have in the docblock.